sandbox/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
     <head>
         <title>Britecharts Sandbox</title>
-        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/britecharts/dist/css/britecharts.min.css" type="text/css" />
+        <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/britecharts/dist/css/britecharts.min.css" type="text/css" /> -->
```

Comment:
Why is this commented out?

Reply:
Hi! I’m glad you like the PR in general.
Regarding this particular line, I have a comment in the main post. 

> Also I've changed how styles are imported in sandbox (see sandbox/index.html and sandbox/sandbox.js changes). Personally I don't care too much about them but I found it's easier to use sandbox if styles are re-compiled dynamically. Let me know if you want to exclude this change completely or if you want me to create another PR for this.

Reply:
Cool, you can remove this. I like the import approach you did in the `sandbox.js`.